content/apps/creating-github-apps/guides/creating-ci-tests-with-the-checks-api.md

```diff
@@ -135,7 +135,7 @@ The `requested` action requests a check run each time code is pushed to the repo
 You'll add this new method as a [Sinatra helper](https://github.com/sinatra/sinatra#helpers) in case you want other routes to use it too. Under `helpers do`, add this `create_check_run` method:
 
 ``` ruby
-# Create a new check run with the status queued
+# Create a new check run with status "queued"
```

Comment:
@skedwards88 thanks for taking a look!

Why the aversion to using quotation marks instead of double-primes?

Reply:
Thanks for asking! I should have included more info. I don't think it would have a major functional impact here since the quotes are in a comment, but stylistically we try to use standard quotation marks (`"`) instead of smart quotes/curly quotes (`”`).